app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -1564,6 +1564,9 @@ function getPluginApi(version) {
       owner.registry.register("plugin-api:main", pluginApi, {
         instantiate: false,
       });
+    } else {
```

Comment:
Small nit: can you swap `if` branches/invert the condition?

Reply:
I'd rather not re-run all of CI just for this change. I think we can live with it for a while.